src/constants/supportedTransforms.ts

```diff
@@ -282,6 +147,16 @@ const supportedTransforms: { [key: string]: string } = {
    */
   original: "orig",
 
+  /**
+   * @link https://docs.imagekit.io/features/image-transformations/image-enhancement-and-color-manipulation#shadow-e-shadow
+   */
+  effectShadow: "e-shadow",
+
+  /**
+   * @link https://docs.imagekit.io/features/image-transformations/image-enhancement-and-color-manipulation.md#gradient-e-gradient
```

Comment:
"https://docs.imagekit.io/features/image-transformations/image-enhancement-and-color-manipulation.md#gradient-e-gradient" broken url

Reply:
done